### ISSUE TITLE
Server SDK local dependency not resolvable by gradle (stitch version 4.4.1)

### ISSUE THREAD

**Issue by robert-cronin** [mongodb/stitch-android-sdk#166]
I am getting a dependency not found error for the Server SDK, it seems Gradle cannot resolve `org.mongodb:stitch-server-services-mongodb-local:4.4.1` which is required by the remote dependency

Here is by `build.gradle`
```
repositories {
    jcenter()
    mavenCentral()
}
...
dependencies {
    implementation 'org.mongodb:stitch-server-core:4.4.1'
    implementation 'org.mongodb:stitch-server-services-mongodb-remote:4.4.1'
}
```
Here is the error message:
```
Could not find org.mongodb:stitch-server-services-mongodb-local:4.4.1.
Searched in the following locations:
  - https://jcenter.bintray.com/org/mongodb/stitch-server-services-mongodb-local/4.4.1/stitch-server-services-mongodb-local-4.4.1.pom
  - https://jcenter.bintray.com/org/mongodb/stitch-server-services-mongodb-local/4.4.1/stitch-server-services-mongodb-local-4.4.1.jar
  - https://repo.maven.apache.org/maven2/org/mongodb/stitch-server-services-mongodb-local/4.4.1/stitch-server-services-mongodb-local-4.4.1.pom
  - https://repo.maven.apache.org/maven2/org/mongodb/stitch-server-services-mongodb-local/4.4.1/stitch-server-services-mongodb-local-4.4.1.jar
Required by:
    project :server > org.mongodb:stitch-server-services-mongodb-remote:4.4.1
```

**Comment by paavithra** [comment]
Getting same issue.. were you able to resolve it

**Comment by robert-cronin** [comment]
nope, I don't think this is resolvable until someone releases version 4.4.1 of the `stitch-server-services-mongodb-local` dependency or at least reduce the dependence to a lower, working version in `server-services-mongodb-remote`

**Comment by adamchel** [comment]
Sorry for missing this earlier, the `stitch-server-services-mongodb-local` package should have been deprecated and removed as a dependency but it was not. I am working on a fix which should be released soon.

**Comment by robert-cronin** [comment]
@adamchel no problems, thanks! I'll keep an eye out

**Comment by adamchel** [comment]
We've just published the SDK version 4.5.0, which should removes the deprecated package entirely and should fix this issue. Please let us know if you run into any other problems.
